src/metatrain/utils/io.py

```diff
@@ -144,12 +148,14 @@ def load_model(
             filename = filename[10:]
         path = hf_hub_download(repo_id, filename, token=kwargs["huggingface_api_token"])
         # make sure to copy the checkpoint to the current directory
-        shutil.copy(path, Path.cwd() / str(path).split("/")[-1])
+        shutil.copy(path, Path.cwd() / filename)
+        logger.info(f"Downloaded model from HuggingFace to {filename}")
```

Comment:
If I understand the code properly, this will download `repo/owner/.../main/some/path/to/file.ckpt` to `<cwd>/some/path/to/file.ckpt`. Is this intended? Otherwise we should be using `os.path.basename(filename)` as the destination